fgrpc/grpcrunner.go

```diff
@@ -168,32 +160,3 @@ func RunGRPCTest(o *GRPCRunnerOptions) (*GRPCRunnerResults, error) {
 	}
 	return &total, nil
 }
-
-// grpcDestination parses dest and returns dest:port based on dest type
-// being a hostname, IP address or hostname/ip:port pair.
-// TODO: change/fix this (NormalizePort and more)
-func grpcDestination(dest string) string {
```

Comment:
can you please leave this code in fgrpc/ - it's specific to grpc no ?

Reply:
It's also network specific and removes the need to import the net std lib. I'll go ahead and move it back if you prefer.

Reply:
Yes I prefer to minimize the diff, it's easier to review, blame etc...
we can move the code later if there is value (but something names "grpcDestination" doesn't belong in fnet/)